server.js

```diff
@@ -236,7 +235,8 @@ function cache(f) {
 
     // Only call vendor servers when last request is older than…
     var cacheInterval = 5000;  // milliseconds
-    var cachedRequest = function (uri, options, callback) {
+    var cachingRequest = function (uri, options, callback) {
+      var request = require('request');
```

Comment:
We only use it twice. It seems clearer to require it both times than introduce a new name. I don't feel that strongly though.